src/index.html

```diff
@@ -9,6 +9,18 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header>
+      <img src="images/logo.png" alt="">
+      <nav>
+        <a class="is-active" href = "#">APPLE</a>
+        <a href = "#">SAMSUNG</a>
+        <a href = "#">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a data-qa="hover" href = "#">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+        <a href = "#">ГАДЖЕТЫ</a>
+        <a href = "#">ПЛАНШЕТЫ</a>
+        <a href = "#">ФОТО</a>
+        <a href = "#">ВИДЕО</a>
```

Comment:
menu items are usually  list items with ul container

Reply:
add uppercase styles in css, and here just usual text with the first upper case letter and lowercase others